API/common/utils.py

```diff
@@ -50,15 +50,13 @@ def execute(cwd, cmd, args=[], quiet=False):
         output = process.communicate()[0]
         exitcode = process.returncode
 
-        if quiet:
-            output = re.sub(' +', ' ', output)
-            output = re.sub('\n\n', '\n', output)
-            output = re.sub('\n ', '\n', output)
```

Comment:
These replacements were required by the `execute('git' ['log', '-1'])` command.
Since that command had modified to have a `--format` option, these steps can be removed.